flow/project.py

```diff
@@ -1388,6 +1391,131 @@ def copy_from(cls, *other_funcs):
 
         return post
 
+    @staticmethod
+    def _setup_operation_register_object(parent_class):
```

Comment:
This name is awkward -- it's long and the words don't tell me very much. I would prefer `_setup_operation_class`, to mirror `_setup_preconditions_class` and `_setup_postconditions_class`.